### PR TITLE
docs: fix inconsistent definition for executor and flow

### PR DESCRIPTION
**Pull request by nan-wang**
# Pull Request Title

## Description

Please describe what your PR is doing and why. Are there any parts which need extra attention during review? Are there any dependencies from other PRs or projcts? Is this a breaking change?

Is this PR ready or work in progress (WIP)? Ready means it can be reviewed and merged from the author's perspective. If the PR is WIP: Make it a draft PR and state open questions and TODO items.

Closes # (issue)

### CODE REVIEW

File: docs/fundamentals/flow/flow-api.md [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 # Flow API
 
-Every `Flow` provides an API to receive requests over the network. Supported protocols are gRPC, HTTP and websockets. We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be send via the Jina Python Client or separate clients like Curl.
+**Flow** ties Executors together into a processing pipeline, provides scalability and facilitates deployments in the cloud. Every `Flow` provides an API to receive requests over the network. Supported protocols are gRPC, HTTP and websockets. 
+
+We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be send via the Jina Python Client or separate clients like Curl. Flow streamlines and scales Executors.
```
Comment (JoanFM) [inline docs/fundamentals/flow/flow-api.md:7]:
```suggestion
We recommend to use gRPC for the best performance, which is also the default protocol. Requests can be sent via the Jina Python Client or separate clients like Curl. Flow streamlines and scales Executors.
```

Comment (JoanFM) [reply docs/fundamentals/flow/flow-api.md:7]: This sentence is weird. 

You define Flow, then add a warning and then keep adding a definition sentence?

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: docs/fundamentals/flow/flow-api.md (hunk 1 above)
Comment (JoanFM) [reply docs/fundamentals/flow/flow-api.md:7]: Also, more than recommending grpc, we should explain the caveats of http

Comment (hanxiao) [reply docs/fundamentals/flow/flow-api.md:7]: merge first , fix it later

**Pull request merged**